app/presenters/publishing_api_presenters/html_attachment.rb

```diff
@@ -0,0 +1,68 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::HtmlAttachment < PublishingApiPresenters::Item
+  def links
+    {
+      parent: [
```

Comment:
I think it'll be safer to call this `parent_document`, as `parent` is used by Finding Things for breadcrumbs and browse pages etc. I don't know for sure that it would conflict, but I think it'd be safer to preempt it. It's also a bit more descriptive. I'm aware this means a change in schemas and frontend code, sorry.

This is just my reckon though, maybe @tijmenb has other ideas about whether this is safe or not?